### PR TITLE
ompss-2 dependencies

### PR DESCRIPTION
**Pull request by viniciusvgp**

### CODE REVIEW

**Review by tldahlgren — APPROVED** [review]
LGTM

File: var/spack/repos/builtin/packages/ompss-2/package.py [hunk 1]
```diff
@@ -31,6 +31,8 @@ class Ompss2(Package):
     depends_on("python", type="build")
     depends_on("cmake", type="build")
     depends_on("extrae", when="+extrae")
+    depends_on("boost@1.59.0:")
+    depends_on("numactl")
```
Comment (tldahlgren) [inline var/spack/repos/builtin/packages/ompss-2/package.py:35]: Do these apply for a specific version on? Or for all listed versions?

**Comment by tldahlgren** [comment]
Deferring merge decision to @dave96 and or @aleixrocks

File: var/spack/repos/builtin/packages/ompss-2/package.py (hunk 1 above)
Comment (viniciusvgp) [reply var/spack/repos/builtin/packages/ompss-2/package.py:35]: Well, I don't know. The documentation (https://pm.bsc.es/ftp/ompss-2/doc/user-guide/build/index.html) lists both as requirements, and I needed this fix to install it (version 2022.11).

Comment (dave96) [reply var/spack/repos/builtin/packages/ompss-2/package.py:35]: They apply to all versions

**Review by dave96 — APPROVED** [review]
LGTM. Thanks @viniciusvgp !

**Comment by aleixrocks** [comment]
LGTM too, thank you very much @viniciusvgp!

**Pull request merged**